tests/Operation/WatchFunctionalTest.php

```diff
@@ -225,6 +226,62 @@ private function assertStartAtOperationTime(TimestampInterface $expectedOperatio
         $this->assertEquals($expectedOperationTime, $command->pipeline[0]->{'$changeStream'}->startAtOperationTime);
     }
 
+    public function testRewindMultipleTimesWithResults()
```

Comment:
These two methods aren't related to resuming, so I included them in their own commit. This is the `rewind()` counterpart to the existing `testNextAdvancesKey` test method.